src/chroma.rs

```diff
@@ -216,7 +216,7 @@ fn chroma_filter(
     Ok(wts.slice_move(s![.., ..non_aliased]))
 }
 
-fn pip_track(
+pub fn pip_track(
```

Comment:
do you think it would be possible to make these functions either `pub(crate)` either to keep them entirely private?

Otherwise they'd show up in the docs and people might think they should be used on their own, while they really shouldn't be touched with a ten-foot pole, they're really bliss' internals :sweat_smile:

Reply:
if it's just pub(crate) then the benchmarks can't see them, but I see your point, how about we do something like this:

```rust
#[cfg(feature = "bench")]
#[inline(always)]
pub fn pip_track_(...) {
    pip_track(...)
}
```

essentially, when the benchmark feature is enabled expose a public wrapper to the inner private functions?

Reply:
The other option is to annotate them with `#[doc(hidden)]`

Reply:
> if it's just pub(crate) then the benchmarks can't see them, but I see your point, how about we do something like this:
> 
> ```rust
> #[cfg(feature = "bench")]
> #[inline(always)]
> pub fn pip_track_(...) {
>     pip_track(...)
> }
> ```
> 
> essentially, when the benchmark feature is enabled expose a public wrapper to the inner private functions?

That sounds like a great idea! Maybe we could put these functions in their own `bench` module / file, and prefix them like e.g. `pub fn bench_pip_track(...) { pip_track(...) }`? Would that work?

Reply:
sure

Reply:
i'll get on that

Reply:
thanks a lot o/